lib/racecar/config.rb

```diff
@@ -10,20 +10,29 @@ class Config < KingKonf::Config
     desc "A string used to identify the client in logs and metrics"
     string :client_id, default: "racecar"
 
-    desc "How frequently to commit offset positions"
-    float :offset_commit_interval, default: 10
```

Comment:
Is there a similar mechanism in rdkafka for setting the offset commit frequency?

Reply:
Yeah, `auto.commit.interval.ms`. I would keep this config.

Reply:
Search for `auto.commit.interval.ms` in https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md (sadly cannot deep link). Defaults to 5 seconds.